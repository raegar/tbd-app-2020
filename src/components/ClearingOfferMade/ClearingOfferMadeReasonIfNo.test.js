import React from 'react';
import { render, cleanup } from '@testing-library/react';
import '@testing-library/jest-dom';
import ClearingOfferMadeThirdPage from './ClearingOfferMadeThirdPage';
import { BrowserRouter as Router} from 'react-router-dom';

//cleans up 
afterEach(cleanup);

it("Renders ClearingOfferMadeThirdPage", () => {
    const { asFragment } = render(<Router><ClearingOfferMadeThirdPage /></Router>);
    expect(asFragment()).toMatchSnapshot();
});