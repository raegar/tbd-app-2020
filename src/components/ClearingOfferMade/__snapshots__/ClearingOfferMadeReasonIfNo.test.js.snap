// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders ClearingOfferMadeReasonIfNo 1`] = `
<DocumentFragment>
  <div
    class="container"
  >
    <p>
      Required
    </p>
    <h1>
      Not Offered Details
    </h1>
    <h3>
      38. If NO, please state reasons:
    </h3>
    <div
      class="mobiletextbox"
    >
      <label />
      <input
        maxlength="30"
        minlength="0"
        placeholder=""
        size="20"
        type="text"
      />
    </div>
    <div
      class="row"
    >
      <div
        class="col"
      >
        <br />
        <a
          href="/ClearingOfferMadeSecondPage"
        >
          <button
            class="mediumbutton"
            data-testid="ucpButton"
            href="/ClearingOfferMadeSecondPage"
            primary="True"
            type="button"
          >
            Go Back​
          </button>
        </a>
      </div>
      <div
        class="col"
      >
        <br />
        <a
          href="/ClearingOfferMadeFifthPage"
        >
          <button
            class="mediumbutton"
            data-testid="ucpButton"
            href="/ClearingOfferMadeFourthPage"
            primary="True"
            type="button"
          >
            Confirm and Continue​
          </button>
        </a>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
